docs/Using-Fleet/REST-API.md

```diff
@@ -5222,17 +5242,24 @@ Returns a list of all queries in the Fleet instance.
-| description      | string | body | The query's description.                                                                                                                               |
-| observer_can_run | bool   | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| Name                            | Type    | In   | Description                                                                                                                                            |
+| ------------------------------- | ------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| name                            | string  | body | **Required**. The name of the query.                                                                                                                   |
+| query                           | string  | body | **Required**. The query in SQL syntax.                                                                                                                 |
+| description                     | string  | body | The query's description.                                                                                                                               |
+| observer_can_run                | bool    | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| team_id                         | integer | body | The parent team to which the new query should be added. If omitted, the query will be global.                                           |
+| interval                       | integer | body | The amount of time, in seconds, the query waits before running. Can be set to `0` to never run. Default: 0.       |
+| platform                        | string  | body | The OS platforms where this query will run (other platforms ignored). Comma-separated string. If omitted, runs on all compatible platforms.                        |
+| min_osquery_version             | string  | body | The minimum required osqueryd version installed on a host. If omitted, all osqueryd versions are acceptable.                                                                          |
+| automations_enabled             | boolean | body | Whether to send data to the configured log destination according to the query's `interval`. |
```

Comment:
If omitted, what's the default value? @rachaelshaw @zhumo

Reply:
Thanks for that! By default should be false.

Reply:
Updated UI interface accordingly